ast/output.go

```diff
@@ -0,0 +1,78 @@
+package ast
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// Output represents the root node of all interpolation evaluations. If the
+// output only has one expression which is either a list or a map, the Output
+// evaluates as a string or a map respectively. Otherwise the Output evaluates
```

Comment:
This should probably read "is type-assertable to a list or a map respectively". Will fix.